op-program/Makefile

```
@@ -26,6 +26,11 @@ op-program-client-mips:
 	# verify output with: readelf -h bin/op-program-client.elf
 	# result is mips32, big endian, R3000
 
+reproducible-prestate:
+	@docker build --output ./bin/ --progress plain -f Dockerfile.repro ../
+	@echo "Absolute prestate hash: $(cat ./bin/prestate-proof.json | jq -r .pre)"
```

For some reason this always prints an empty string for me. The `$(...)` syntax doesn't work like you expect because it first gets interpreted by make as a variable replacement (I think).

I'd suggest:
```suggestion
	@echo "Absolute prestate hash:"
	@jq -r .pre ./bin/prestate-proof.json
```